SharedSupport/Default Bundles/Format SQL.spBundle/command.plist

```diff
@@ -9,22 +9,1099 @@
+     * @param String $value The token's value
+     *
+     * @return String HTML code of the highlighted token.
+     */
+    protected static function highlightError($value)
+    {
+        if (self::is_cli()) {
+            return self::$cli_error . $value . "\x1b[0m";
+        } else {
+            return '&lt;span ' . self::$error_attributes . '&gt;' . $value . '&lt;/span&gt;';
+        }
+    }
+
+    /**
+     * Highlights a comment
+     *
+     * @param String $value The token's value
+     *
+     * @return String HTML code of the highlighted token.
+     */
+    protected static function highlightComment($value)
+    {
+        if (self::is_cli()) {
+            return self::$cli_comment . $value . "\x1b[0m";
+        } else {
+            return '&lt;span ' . self::$comment_attributes . '&gt;' . $value . '&lt;/span&gt;';
+        }
+    }
+
+    /**
+     * Highlights a word token
+     *
+     * @param String $value The token's value
+     *
+     * @return String HTML code of the highlighted token.
+     */
+    protected static function highlightWord($value)
+    {
+        if (self::is_cli()) {
+            return self::$cli_word . $value . "\x1b[0m";
+        } else {
+            return '&lt;span ' . self::$word_attributes . '&gt;' . $value . '&lt;/span&gt;';
+        }
+    }
+
+    /**
+     * Highlights a variable token
+     *
+     * @param String $value The token's value
+     *
+     * @return String HTML code of the highlighted token.
+     */
+    protected static function highlightVariable($value)
+    {
+        if (self::is_cli()) {
+            return self::$cli_variable . $value . "\x1b[0m";
+        } else {
+            return '&lt;span ' . self::$variable_attributes . '&gt;' . $value . '&lt;/span&gt;';
+        }
+    }
+
+    /**
+     * Helper function for building regular expressions for reserved words and boundary characters
+     *
+     * @param String $a The string to be quoted
+     *
+     * @return String The quoted string
+     */
+    private static function quote_regex($a)
+    {
+        return preg_quote($a,'/');
+    }
+
+    /**
+     * Helper function for building string output
+     *
+     * @param String $string The string to be quoted
+     *
+     * @return String The quoted string
+     */
+    private static function output($string)
+    {
+        if (self::is_cli()) {
+            return $string."\n";
+        } else {
+            $string=trim($string);
+            if (!self::$use_pre) {
+                return $string;
+            }
+
+            return '&lt;pre '.self::$pre_attributes.'&gt;' . $string . '&lt;/pre&gt;';
+        }
+    }
+
+    private static function is_cli()
+    {
+        if (isset(self::$cli)) return self::$cli;
+        else return php_sapi_name() === 'cli';
+    }
+
+}
+
+$sql = @file_get_contents("php://stdin");
+echo trim(SqlFormatter::format($sql, false));</string>
 	<key>contact</key>
 	<string></string>
+	<key>defaultBundleWasModified</key>
+	<true/>
```

Comment:
Perhaps this is a key from debug? I believe this should be dropped from the plist